packages/contracts/deploy/helpers.ts

```diff
@@ -215,12 +184,58 @@ export async function createPluginRepo(
     pluginRepoFactoryAddress
   );
 
-  const {deployer} = await hre.getNamedAccounts();
+  const pluginDomain =
+    process.env[`${network.name.toUpperCase()}_PLUGIN_ENS_DOMAIN`];
+
+  const node = ethers.utils.namehash(`${pluginName}.${pluginDomain}`);
+
+  const pluginSubdomainRegistrar = await getContractAddress(
+    'Plugin_ENSSubdomainRegistrar',
+    hre
+  );
+
+  const ensRegistryContract = ENSRegistry__factory.connect(
+    await getENSAddress(hre),
+    signers[0]
+  );
+
+  const owner = await ensRegistryContract.owner(node);
+
+  // ENS subdomain has already been registered by pluginSubdomainRegistrar.
+  // Running the `createPluginRepo` will fail unless skipped.
+  // Since `aragonPluginRepos` is filled in run-time, we still need to refill it.
+  if (owner === pluginSubdomainRegistrar) {
+    const pluginRepoRegistryFactory = new PluginRepoRegistry__factory(
+      signers[0]
+    );
+    const pluginRepoRegistry = pluginRepoRegistryFactory.attach(
+      await getContractAddress('PluginRepoRegistry', hre)
+    );
+    let events = await pluginRepoRegistry.queryFilter(
+      pluginRepoRegistry.filters.PluginRepoRegistered(null, null)
+    );
+    const found = events.filter(event => event.args.subdomain == pluginName);
```

Comment:
```suggestion
    const found = events.filter(event => event?.args?.subdomain == pluginName);
```
Better to avoid JS blowups